takwimu/templates/takwimu/_includes/report/country_selector.html

```diff
@@ -1,13 +1,25 @@
 {% load staticfiles %}
 
+{% block content %}
+<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
 <div class="btn-group">
   <button class="btn btn-outline-secondary btn-lg dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
     <img class="rounded-circle border-light mr-2" src="{% static 'img/flags/nigeria.svg' %}"
         border="8" style="border-style:double; width: 50px; height: 50px;"/>
     Nigeria
   </button>
-  <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-    <a class="dropdown-item" href="#">Senegal</a>
-    <a class="dropdown-item" href="#">Tanzania</a>
+  <div class="dropdown-menu pt-0 rounded-0" aria-labelledby="dropdownMenuButton" style="width: 180px;">
+
+    <a class="dropdown-item" href="#" data-value="senegal"><img class="rounded-circle border-light mr-2" src="{% static 'img/flags/senegal.svg' %}" border="8" style="border-style:double; width: 50px; height: 50px;"/>Senegal</a>
+    <a class="dropdown-item" href="#" data-value="tanzania"><img class="rounded-circle border-light mr-2" src="{% static 'img/flags/tanzania.svg' %}" border="8" style="border-style:double; width: 50px; height: 50px;"/>Tanzania</a>
+    <a class="dropdown-item" href="#" data-value="tanzania"><img class="rounded-circle border-light mr-2" src="{% static 'img/flags/nigeria.svg' %}" border="8" style="border-style:double; width: 50px; height: 50px;"/>Nigeria</a>
+
   </div>
-</div>
\ No newline at end of file
+</div>
+<!--Enable dropdown select list-->
+<script>
+	$('.dropdown-menu a').click(function(){
+		$(".btn:first-child").html($(this).html()+' <span class="caret"></span>');
+	});
+</script>
```

Comment:
You'll want to do this in a `window.onload` function to wait for jQuery to be available.

Reply:
@DavidLemayian Made the necessary changes and places the scripts in the base layout template